src/pretalx/orga/forms/event.py

```diff
@@ -101,6 +101,11 @@ class EventSettingsForm(ReadOnlyFlag, I18nFormMixin, HierarkeyForm):
         help_text=_('Unset to hide your schedule, e.g. if you want to use the HTML export exclusively.'),
         required=False,
     )
+    show_sneak_peek = forms.BooleanField(
+        label=_('Show a sneak peek before schedule release'),
+        help_text=_('Set to publicly display a list of talks, which have the "is featured" flag enabled.'),
```

Comment:
"Set to display a public list of talks prior to a schedule release. You can configure the talks in the … area."